beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/forward/multipeer/BatchDataRequesterTest.java

```diff
@@ -29,13 +29,15 @@
 import tech.pegasys.teku.beacon.sync.forward.multipeer.chains.TargetChainTestUtil;
 import tech.pegasys.teku.infrastructure.async.eventthread.InlineEventThread;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.TestSpecFactory;
 import tech.pegasys.teku.spec.datastructures.blocks.SlotAndBlockRoot;
 import tech.pegasys.teku.spec.util.DataStructureUtil;
 
 class BatchDataRequesterTest {
   private static final UInt64 BATCH_SIZE = UInt64.valueOf(50);
   private static final int MAX_PENDING_BATCHES = 5;
-  private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+  private final DataStructureUtil dataStructureUtil =
+      new DataStructureUtil(TestSpecFactory.createMinimalPhase0());
```

Comment:
We probably should use `TestSpecFactory.createDefault()` for these since they generally don't care what the spec actually is.